lib/rucio/web/rest/flaskapi/v1/auth.py

```diff
@@ -107,17 +112,30 @@ def get(self):
         appid = request.headers.get('X-Rucio-AppID', default='unknown')
         ip = request.headers.get('X-Forwarded-For', default=request.remote_addr)
 
-        if not account or not username or not password:
+        if not username or not password:
             return generate_http_error_flask(401, CannotAuthenticate.__name__, 'Cannot authenticate without passing all required arguments', headers=headers)
 
+        if not account:
```

Comment:
I think we still need to check the validity of username & password. This way, if you do not give an `account` you can access the list of accounts the username has access to, without having a valid password. This is an issue.

The second problem I see here is also MultiVO deployments. The `list_accounts_for_identity` method as well as `get_default_account` would need to be made Multi VO compatible. @Thysk any input?